tools/Makefile

```diff
@@ -5,7 +5,7 @@ GCC_VERSION := $(shell gcc -dumpversion)
 
 .PHONY: all clean
 
-all: venv chainer_patch.done kaldi.done nkf.done warp-ctc.done chainer_ctc.done sentencepiece.done kaldiio moses
+all: venv chainer_patch.done kaldi.done nkf.done warp-ctc.done chainer_ctc.done sentencepiece.done kaldiio moses nltk
```

Comment:
It's better to add the `nltk` installation to https://github.com/espnet/espnet/blob/v.0.3.0/setup.py instead of Makefile